source/common/access_log/BUILD

```diff
@@ -18,7 +18,7 @@ envoy_cc_library(
     deps = [
         "//include/envoy/access_log:access_log_interface",
         "//include/envoy/config:typed_config_interface",
-        "//include/envoy/filesystem:filesystem_interface",
+        # "//include/envoy/filesystem:filesystem_interface",
```

Comment:
already fixed but waiting to push on the next commit so that I don't interrupt the CI